nginx-proxy/imgpath.js

```diff
@@ -0,0 +1,5 @@
+function file_path(r) {
+    var uri = decodeURI(r.uri);
+    var digest = require('crypto').createHash('sha1').update(uri).digest('hex');
+    return digest.slice(NaN, 2) + "/" + digest.slice(2, 4) + "/" + digest.slice(4);
```

Comment:
maybe NaN -> 0 ?

Reply:
No, according to the official docs it should be NaN https://nginx.org/en/docs/njs/reference.html (search for the slice)